artefacts/README.md

```diff
@@ -1 +1,16 @@
+[<< Back](https://cntt-n.github.io/CNTT/)
+# Handling of figures
 
+## Table of Contents
+
+* [1 Usage of figures](#1)
+
+<a name="1"></a>
+## 1 Usage of figures
+
+- Figures used in the document should be in Portable Network Graphic (PNG) format
+- Figures must be stored in the `figures` folder of the document where the figures are used (e.g.: `doc/ref_arch/figures/`)
+- File name of the figures shold contain the chapter number where the figure is used as a prefix
+- Every figure used in the documents should have an editable version commited to the repository in `ppt(x)` or in `svg` format
```

Comment:
If we're adding svg support, can Github support the display of svg natively (and thus not need to have a .png format generated separately from the artifacts?)

Reply:
Hopefully corrected, but please check @walterkozlowski -s comment and my question about removing the svg support.